toolchains/cross_compiler/cc-toolchain-config.bzl

```diff
@@ -175,13 +186,36 @@ def _impl(ctx):
             ),
             flag_set(
                 actions = [ACTION_NAMES.c_compile],
-                flag_groups = [],
+                flag_groups = ([
+                    flag_group(
+                        flags = [
+                            "-Wformat=2",
+                            "-pedantic",
+                            "-DAUSTIN_TOOLCHAIN",
```

Comment:
Probably don't want this

Reply:
Lol, and this is why we do code review.  Thanks.